.github/CODEOWNERS

```diff
@@ -195,6 +195,7 @@ receiver/skywalkingreceiver/                         @open-telemetry/collector-c
 receiver/snmpreceiver/                               @open-telemetry/collector-contrib-approvers @djaglowski @StefanKurek @tamir-michaeli
 receiver/solacereceiver/                             @open-telemetry/collector-contrib-approvers @djaglowski @mcardy
 receiver/splunkhecreceiver/                          @open-telemetry/collector-contrib-approvers @atoulme @keitwb
+receiver/snowflakereceiver/                          @open-telemetry/collector-contrib-approvers @dmitryax
```

Comment:
Please add your handle if you can support if going forward as well

Reply:
fixed!